--- libextractor-python-0.5.4/Makefile.am.orig	2005-08-27 08:19:21.000000000 +0200
+++ libextractor-python-0.5.4/Makefile.am	2005-08-29 22:16:36.091081680 +0200
@@ -1,9 +1,9 @@
 all:
 	python libextractor_python_setup.py $(EXT_LIB_PATH) build
 install-exec-local:
-	python libextractor_python_setup.py $(EXT_LIB_PATH) install --prefix=$(PYTHON_PREFIX)
+	python libextractor_python_setup.py $(EXT_LIB_PATH) install --prefix=$(DESTDIR)$(PYTHON_PREFIX)
 EXTRA_DIST = \
   extract.py \
   libextractor_python_setup.py \
   libextractor_python.c \
-  README.debian
\ Brak znaku nowej linii na koñcu pliku
+  README.debian
